src/Components/Events.js

```diff
@@ -17,38 +21,60 @@ function Events() {
             </button>
           </a>
         ) : (
-          <p className="event-status">{event.publish ? (event.end ? "This event has ended." : "") : "RSVP releasing soon✨"}</p>
+          <p className="event-status">
+            {event.publish
+              ? event.end
+                ? "This event has ended."
+                : ""
+              : "RSVP releasing soon✨"}
+          </p>
         )}
       </div>
     </div>
   ));
 
-  const postId = ["C2-o7roPbfN", "C23PyAWPYdW", "C2boAR4LeRy", "CzVNKp6r9hn", "CzVLbJWLwsI", "Cy4WTIJvVvB", "CyzVzMTJlhQ", "CyRondZPVVS", "CyCms3mLOLb", "Cxum20yOlkQ", "CxrxWcQLpSb"];
+  const postId = [
```

Comment:
Is there a reason for hardcoding postIds? Are we not just wanting the most recent `n` posts? Maybe you can't answer this but maybe @elimelt can?